lib/internal/bootstrap/loaders.js

```diff
@@ -281,6 +282,8 @@ class BuiltinModule {
   getESMFacade() {
     if (this.module) return this.module;
     const { ModuleWrap } = internalBinding('module_wrap');
+    // TODO(aduh95): move this to C++, alongside the initialization of the class.
+    ObjectSetPrototypeOf(ModuleWrap.prototype, null);
```

Comment:
The initialization is done here https://github.com/nodejs/node/blob/4eaaa1755222a4e167b74d1ca4c467e69c55cca9/src/module_wrap.cc#L763-L773

I couldn't find a way to change the prototype from there. @RaisinTen do you know how to do that by any chance?

Reply:
@aduh95 I spoke to @LeszekSwirski about this on the Chromium Slack and it seems like this is not possible to do using V8's public API, so I've opened a feature request asking for this - https://bugs.chromium.org/p/v8/issues/detail?id=13392.

Reply:
OK bummer. Thanks a lot for looking into this and opening the feature request issue :)